.github/workflows/ci.yml

```diff
@@ -38,6 +38,15 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Install Chromium for Selenium testing
+      run: |
+        sudo apt update
+        sudo apt install snapd
+        sudo snap install chromium
+        wget https://chromedriver.storage.googleapis.com/93.0.4577.63/chromedriver_linux64.zip
+        sudo unzip chromedriver_linux64.zip -d /usr/local/bin/
+        sudo chmod +x /usr/local/bin/chromedriver
+
```

Comment:
We don't need to add this. It is [already added in the virtual environment](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#browsers-and-drivers) used by GitHub Actions.